### DISCUSSION TITLE
How to apply styles to & disable tiles for certain dates?

### DISCUSSION THREAD

**Discussion by AndreYanny** [wojtekmaj/react-calendar#610] [Q&A]
I'm making a booking website. I made an array called bookedDates, that's filled with dummy dates. Now I want to change the background color of the booked dates and if possible make them inaccessible for anyone who wants to make any bookings in those dates.

This is my Calendar code:

const [date, onChange] = useState(new Date());

<Calendar
onChange={onChange}
value={date}
selectRange={true}
/>

**Comment by wojtekmaj** [comment]
This is how you can apply tile styles based on list of dates:

https://codesandbox.io/s/react-calendar-tileclassname-0nlmh

and almost identically, you can disable tiles based on list of dates:

https://codesandbox.io/s/react-calendar-tiledisabled-v7xfuh
